src/index.html

```diff
@@ -8,11 +8,49 @@
     />
     <title>Calendar</title>
     <link
-      rel="stylesheet"
-      href="styles/index.scss"
+      rel="styles"
+      href="styles/index.css"
     />
   </head>
+
   <body>
     <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__month calendar__month--length-31">
+        <div class="calendar__day calendar__day--start-sun">
```

Comment:
The `calendar__day` element is used here both as a container for days and as a day itself which is semantically incorrect. Each day should be an instance of `calendar__day`, and they should be contained within a separate element that represents the grid or list of days.